narui_core/src/vulkano_render/lyon.rs

```diff
@@ -77,11 +77,12 @@ impl<'a> GeometryBuilder for NaruiGeometryBuilder<'a> {
         self.index_offset = self.data.indices.len() as _;
     }
 
-    fn end_geometry(&mut self) -> Count {
-        Count {
-            vertices: self.data.vertices.len() as u32 - self.vertex_offset,
-            indices: self.data.indices.len() as u32 - self.index_offset,
-        }
+    fn end_geometry(&mut self) // -> Count
+    {
+        // Count {
```

Comment:
lets not uncomment this?

Reply:
well there is no return value, I am not really sure what one is supposed to do in this function....

Reply:
Probably just delete it?